tests/integration/scenarios/out_syslog/tests/test_out_syslog_001.py

```
@@ -0,0 +1,267 @@
+        return self.message
+
+
+class DtlsReceiver:
+    def __init__(self, port, cert_file, key_file):
+        self.port = port
+        self.cert_file = cert_file
+        self.key_file = key_file
+        self.process = None
+
+    def start(self):
+        self.process = subprocess.Popen(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)